components/Layout/NeedHelpList.tsx

```diff
@@ -1,19 +1,62 @@
+import { Instance } from 'mobx-state-tree'
+import { useEffect } from 'react'
+import { SummaryLink } from '../../client-state/store'
+import { useMediaQuery } from '../Hooks'
+
 export const NeedHelpList: React.VFC<{
   title: string
-  links: { label: string; location: string }[]
-}> = ({ title, links }) => (
-  <div>
-    <div className="p-8 bg-emphasis rounded">
-      <h2 className="h2">{title}</h2>
-      <ul className="pl-5 list-disc">
-        {links.map(({ label, location }, index) => (
-          <li key={index}>
-            <a href={location} className="text-default-text underline">
-              {label}
+  links: Instance<typeof SummaryLink>[]
+}> = ({ title, links }) => {
+  const isMobile = useMediaQuery(992)
+
+  // handler on scroll for sticky need help
+  const shouldBeSticky = (e) => {
+    const self = document.querySelector('.fixedElement')
+
+    // do not add this behaviour on mobile
+    if (isMobile) return
+
+    const scrollTop = window.scrollY
+    scrollTop >= 700
+      ? self.classList.add('is-sticky')
+      : self.classList.remove('is-sticky')
+  }
+
+  useEffect(() => {
+    window.addEventListener('scroll', shouldBeSticky)
+    return () => {
+      window.removeEventListener('scroll', shouldBeSticky)
+    }
+  })
+
+  return (
+    <div className="fixedElement">
+      <div className="p-8 bg-emphasis rounded mt-8 md:mt-0">
+        <h2 className="h2">{title}</h2>
+        <ul className="pl-5 list-disc">
+          <li>
+            <a
+              className="text-default-text underline"
+              href="https://www.canada.ca/en/employment-social-development/corporate/contact/oas.html"
+              target="_blank" rel="noreferrer"
+            >
+              Contact Service Canada
```

Comment:
This link will be coming from the backend, I've made the change already in another branch, and just cherry-picked it into this one. So you can remove the SC link from your code and it'll come along with the rest of the links.

However, I noticed something a bit weird when testing this, on initial load, there is no request to the backend, so you don't use my links until there is user input. This is also causing an issue where, when you switch to Results immediately after first page load, you see this:
![image](https://user-images.githubusercontent.com/3630698/151223202-1900c6ba-0e57-40d8-80ac-39195f0f9617.png)

Reply:
There should be a request to the backend, but I think the summary object is not being set. I think I have a fix in mind!

Reply:
Confirmed fixed!